### DISCUSSION TITLE
Is there any way to adjust font sizes currently?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#819] [Q&A] [answered]
I assume somewhere a font family, size etc is set... wondering if this is changeable?

**Comment by mtanco** [comment, marked as answer]
At this point font details and css are kept as defaults, however you can use the following UI elements for different relative sizes:
`ui.text_xs, ui.text_s, ui.text_m, ui.text_l, ui.text_xl`

**Reply by Alex-Wenner-FHR** [reply]
Would this work on a component such as a table?

**Reply by mtanco** [reply]
Ah, no not today, only as individual UI components
